tests/stdlib/tmath.nim

```diff
@@ -106,6 +106,27 @@ template main() =
     doAssert euclDiv(-9, -3) == 3
     doAssert euclMod(-9, -3) == 0
 
+  block: # ceilDiv/fastCeilDiv
+    doAssert ceilDiv(8, 3) == 3
+    doAssert ceilDiv(8, -3) == -2
+    doAssert ceilDiv(-8, 3) == -2
+    doAssert ceilDiv(-8, -3) == 3
+    doAssert ceilDiv(8, 4) == 2
+    doAssert ceilDiv(8, -4) == -2
+    doAssert ceilDiv(-8, 4) == -2
+    doAssert ceilDiv(-8, -4) == 2
+    doAssert ceilDiv(11,  3) ==  4
+    doAssert ceilDiv(12,  3) ==  4
+    doAssert ceilDiv(13,  3) ==  5
+
+    doAssert fastCeilDiv(8,  3) ==  3
+    doAssert fastCeilDiv(8,  4) ==  2
+    doAssert fastCeilDiv(8,  5) ==  2
+    doAssert fastCeilDiv(11,  3) ==  4
+    doAssert fastCeilDiv(12,  3) ==  4
+    doAssert fastCeilDiv(13,  3) ==  5
+    doAssert fastCeilDiv(41,  7) ==  6
```

Comment:
```suggestion
    doAssertRaises(AssertionDefect): discard fastCeilDiv(41,  0)
    doAssertRaises(AssertionDefect): discard fastCeilDiv(41,  -1)
    doAssertRaises(AssertionDefect): discard fastCeilDiv(-1,  1)
```

Reply:
@timotheecour 
Thank you! I have fixed the code.